libbeat/esclientleg/url.go

```diff
@@ -74,8 +74,7 @@ func makePath(index string, docType string, id string) (string, error) {
 	return path, nil
 }
 
-// TODO: make this reusable. Same definition in elasticsearch monitoring module
-func parseProxyURL(raw string) (*url.URL, error) {
+func ParseProxyURL(raw string) (*url.URL, error) {
```

Comment:
hm... this function looks quite generic. We might have some similar function already in libbeat.

Reply:
Indeed, I will look or make one and remove this one.

Reply:
I could not find an existing one so I made a new one in the `common` package: 81621350c.